server/channels/store/sqlstore/store_test.go

```diff
@@ -795,6 +795,50 @@ func TestReplicaLagQuery(t *testing.T) {
 	}
 }
 
+func TestInvalidReplicaLagDataSource(t *testing.T) {
+	logger := mlog.CreateConsoleTestLogger(t)
+
+	testDrivers := []string{
+		model.DatabaseDriverPostgres,
+		model.DatabaseDriverMysql,
+	}
+
+	for _, driver := range testDrivers {
+		t.Run(driver, func(t *testing.T) {
+			settings, err := makeSqlSettings(driver)
+			if err != nil {
+				t.Skip(err)
+			}
```

Comment:
Not something to fix for this PR, but:
I see that `t.Skip(err)` is widely used in other tests after `makeSqlSettings`, but why don't we simply do `require.NoError(t, err)`?

Reply:
Hmm .. I don't see a reason for that :thinking: